src/clients/python/samples/basic/README.md

```diff
@@ -0,0 +1,61 @@
+<!-- This file is generated by [/src/scripts/client_readmes.zig](/src/scripts/client_readmes.zig). -->
+# Basic Python Sample
+
+Code for this sample is in [./main.py](./main.py).
+
+## Prerequisites
+
+Linux >= 5.6 is the only production environment we
+support. But for ease of development we also support macOS and Windows.
+* Python (or PyPy, etc) >= `3.7`
+
+## Setup
+
+First, clone this repo and `cd` into `tigerbeetle/src/clients/python/samples/basic`.
+
+Then, install the TigerBeetle client:
+
+```console
+pip install tigerbeetle_python
```

Comment:
To confirm this the name we want to go with - not eg just `tigerbeetle`.

Reply:
After chatting with Joran, we're going to go the opposite direction! All clients will just be `tigerbeetle` in their respective languages. I'll hold off renaming this one though, until you're done with review.